server.js

```diff
@@ -1,35 +1,123 @@
+      console.error("Error seeding database:", error);
+    }
+  };
+  seedDatabase();
+}
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl);
-mongoose.Promise = Promise;
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({
+    message: "Welcome to the Golden Globes API!",
+    endpoints: {
+      "/golden-globes": "Get all Golden Globes nominations",
+      "/golden-globes/winners": "Get all winners",
+      "/golden-globes/category/:category": "Get nominations by category",
+      "/golden-globes/nominee/:nominee": "Get nomination by nominee",
+    },
+  });
+});
+
+app.get("/golden-globes", async (req, res) => {
+  try {
+    const nominations = await GoldenGlobe.find();
+    res.json(nominations);
+  } catch (error) {
+    console.error("Error fetching nominations:", error);
+    res.status(500).json({ error: "Internal server error" });
+  }
+});
+
+app.get("/golden-globes/winners", async (req, res) => {
+  try {
+    const winners = await GoldenGlobe.find({ win: true });
+    res.json(winners);
+  } catch (error) {
+    console.error("Error fetching winners:", error);
+    res.status(500).json({ error: "Internal server error" });
+  }
+});
+
+app.get("/golden-globes/category/:category", async (req, res) => {
+  const { category } = req.params;
+  try {
+    const nominees = await GoldenGlobe.find({ category });
+    if (nominees.length > 0) {
+      res.json(nominees);
+    } else {
+      res.status(404).json({ error: "No nominations found for this category" });
+    }
+  } catch (error) {
+    console.error(`Error fetching nominations for category: ${category}`, error);
+    res.status(500).json({ error: "Internal server error" });
+  }
+});
+
+app.get("/golden-globes/nominee/:nominee", async (req, res) => {
+  const { nominee } = req.params;
+  try {
+    const result = await GoldenGlobe.findOne({ nominee });
+    if (result) {
+      res.json(result);
+    } else {
+      res.status(404).json({ error: "Nominee not found" });
+    }
+  } catch (error) {
+    console.error(`Error fetching nominee: ${nominee}`, error);
+    res.status(500).json({ error: "Internal server error" });
+  }
 });
```

Comment:
Remember to use query params for filters to make it more RESTful. Endpoints should be named after what they return, and you're not returning categories for example, you're returning nominations or golden-globes _filtered_ on a category